cvat-core/src/server-proxy.ts

```diff
@@ -481,13 +493,16 @@ async function getSelf() {
 
 async function authorized() {
     try {
+        // In CVAT app we use two types of authentication
+        // At first we check if authentication token is present
+        // Request in getSelf will provide correct authentication cookies
+        if (!store.get('token')) {
+            await logout();
```

Comment:
Can we just remoive cookies? 
There is one big difference between server ``logout`` and removing cookies from the client.
In the first case we also logout all the authorized sessions on the server. Do you think it is expected behaviour?

Reply:
We can't just remove cookies. `sessionid` is protected by `HttpOnly` flag which prevents browser from reading/writing this cookie.
But I would agree that server logout is overkill in that situation, even if we cant remove cookie we can still redirect user to login page, so if he logins once again his session would be restored and no logout needed.